innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+* InnerSource Portal
+
+## Patlet
+
+InnerSource projects at your company are increasing but potential contributors have no easy way of discovering them.
+
+Create an intranet website that indexes all available InnerSource projects.  Make it easy for InnerSource projects owners to include information about their projects and for visitors to the site to search across all the available projects.  This will enable potential contributors to more easily locate projects of interests and learn more about who is running them, the project goals and access technical and community management related documentation.
+
+## Problem
+
+There is no central, easily accessible resource for InnerSource project owners and potential contributors to leverage to advertise or discover InnerSource projects.  
```

Comment:
The actual problem is missing here. What is the result of the missing platform?

Reply:
OK. I'll add a single line defining the problem before suggesting a solution.  Right now they are combined together in to this one sentence.